browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1661,7 +1664,7 @@ BraveRewardsPageUI::BraveRewardsPageUI(
     content::WebUI* web_ui,
     const std::string& name)
     : BasicUI(web_ui, name,
-#if !defined(OS_ANDROID)
+#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
same as above, this is very strange to put here

Reply:
I explained it here https://github.com/brave/brave-core/pull/3731/files#r336516733

Reply:
ok